lib/rubocop/cop/rspec/base.rb

```diff
@@ -68,6 +68,12 @@ def rspec_pattern_config
             .fetch('RSpec', DEFAULT_CONFIGURATION)
             .fetch('Patterns')
         end
+
+        def rspec_language_config
+          all_cops_config
+            .fetch('RSpec', DEFAULT_CONFIGURATION)
+            .fetch('Language')
+        end
```

Comment:
Maybe push this down to `Language`? This way `ExampleGroup`/`Concept` will also have access to it, and we won't have to pass `rspec_language_config` as an argument?

Reply:
Would be cool, but how can I access `config` from outside of cop?

Reply:
Fair enough, there's no way.